library/src/test/java/org/apache/apex/malhar/lib/state/spillable/SpillableMapImplTest.java

```diff
@@ -31,31 +33,46 @@
 import com.datatorrent.lib.helper.OperatorContextTestHelper;
 import com.datatorrent.lib.util.KryoCloneUtils;
 
+import junitparams.JUnitParamsRunner;
```

Comment:
I think we should use classes under org.junit. WindowedOperatorTest uses:
{code}
import org.junit.runner.RunWith;
import org.junit.runners.Parameterized;
{code}

etc.